.github/workflows/track-no-cover.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: Check no cover Removal
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  "15 9 1 * *"
+
+jobs:
+  check-no-cover:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: "actions/setup-python@v2"
+        with:
+          python-version: "3.9"
+      - uses: extractions/setup-just@v1
+
+      - name: "Stop ignoring pragma: no cover lines"
+        run: |
+          set -eu
+          sed -i.bak 's/"pragma: no cover",/# "pragma: no cover",/' pyproject.toml
+
+      - name: Did coverage change?
+        env:
+          CI: 1
+        run: |
+          set -eu
+          set -o pipefail
+
+          just test | tee test-output.txt || true
+
+          # extract missing lines per file fom coverage report in the tests output
+          # python script: get the files table from coverage
+          # awk: break the table down into filename and missing lines number
+          python scripts/parse_coverage_output.py test-output.txt | awk '{ print $1 " " $3 }' - > no-cover.txt
```

Comment:
Seems a bit odd that we're using `awk` to parse the output of a script that we've written expressly for this purpose rather than getting it to output the format we actually want.

Reply:
Right tool for the job, etc.  It's far easier/quicker to grab tokens 1&3 with awk than do all the breaking down and stripping of leading/trailing spaces in Python.